SConstruct

```diff
@@ -0,0 +1,3 @@
+# -*- python -*-
+from lsst.sconsUtils import scripts
+scripts.BasicSConstruct("modelfit_parameters", versionModuleName=None)
```

Comment:
Without that, scons generated a `python/lsst/modelfit/parameters/version.py` file that I don't think I want, since there is no Python module.

Having said that, I do have to resolve how to do versioning. With some difficulty I got meson to read a static `VERSION` file but the docs do say to ask in #dm-arch if attempting to use semver along with eups versioning and I'm not sure what the best choice is here, so I will do that.

Reply:
> the docs do say to ask in #dm-arch if attempting to use semver along with eups versioning

Can you point me to the link? I didn't realize there was a policy on that.

Reply:
It's in the [page on making packages pip-installable](https://developer.lsst.io/stack/building-with-pip.html), most of which isn't relevant here but the versioning parts are.